Formula/lensfun.rb

```diff
@@ -30,12 +19,6 @@ class Lensfun < Formula
   depends_on "libpng"
   depends_on "python@3.9"
 
-  # This patch can be removed when new Lensfun release (v0.3.96) is available.
```

Comment:
Is h this also fixed in 0.3.3?

Reply:
0.3.2 did not have this issue. As 0.3.3 is based upon 0.3.2 there is no need to patch it. Or in other words this patch is specific to 0.3.95 and not required for 0.3.3.